src/sql/workbench/contrib/queryplan2/browser/queryPlan.ts

```diff
@@ -194,7 +191,12 @@ export class QueryPlan2 implements ISashLayoutProvider {
 			if (newHeight < 200) {
 				return;
 			}
-			this._container.style.height = `${newHeight}px`;
+			/**
+			 * Since the parent container is flex, we will have
+			 * to change the flex-basis property to change the height.
+			 */
+			this._container.style.minHeight = '200px';
```

Comment:
This seems like it should belong in the css

Reply:
This overrides CSS's min-width. Intially, the min-width is set to 500px. After resizing, the min-width is updated to 200px.

Reply:
I'm not really understanding what this is doing then - why 200px? And why only do this after resizing?

Reply:
Since we are trying to equally divide the tab container into query plans, we must set the min-height so that query plans do not become too short. It prevents situations like this. (when there are many query plans, they become too short)
![image](https://user-images.githubusercontent.com/6816294/154358653-3bd3034b-8579-4e22-89fa-f3f72cd3b496.png)

We fix this issue by changing the min-height to 500px
![image](https://user-images.githubusercontent.com/6816294/154358813-8edef95c-6391-4447-b504-eb1a6631a47a.png)


Now, if the user wants to manually drag and resize the queryplans, the min-height of 500px will prevent them from doing so. Therefore, as soon as the user drags the sash, we change the min-height from 500px to 200px. 
![image](https://user-images.githubusercontent.com/6816294/154358875-c3d116e5-9a4a-4aa1-894e-2d9d18249582.png)

Reply:
200px is an arbitrary number. There is no good reason behind it other than the query plan starts looking bad when we go below that.

Reply:
But why not just have a min-height of 200px to start with then? Or just set the initial height to 500px if you want to start off with it being bigger but allow them to make it smaller.

Reply:
The problem I was there was setting the initial height to 500px will prevent it from taking up the entire space.